frontend/app/module/input/page/FinaliseElectionPage.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { RouterProvider } from "react-router-dom";
+
+import { render as rtlRender, screen } from "@testing-library/react";
+import { describe, expect, test, vi } from "vitest";
+
+import { expectNotFound, overrideOnce, render, setupTestRouter } from "app/test/unit";
+
+import { ApiProvider, ElectionProvider, ElectionStatusProvider } from "@kiesraad/api";
+
+import { FinaliseElectionPage } from "./FinaliseElectionPage";
+
+describe("FinaliseElectionPage", () => {
+  test("Error when election is not ready", async () => {
+    // Since we test what happens after an error, we want vitest to ignore them
+    vi.spyOn(console, "error").mockImplementation(() => {
+      /* do nothing */
+    });
+    const router = setupTestRouter();
+
```

Comment:
```suggestion

    overrideOnce("get", "/api/elections/1/status", 200, {
      statuses: [
        { id: 1, status: "Incomplete" },
        { id: 2, status: "Complete" },
      ],
    });

```

Reply:
Why would we do this override? This equals the response as defined in de MSW

Reply:
To be explicit about what determines the behavior of what is being tested, so it makes the tests more readable.

For example three years down the line, when we have a new dev just joining the team. They can then immediately see it's the input statuses not being complete that's key, instead of having to look at the code of the page, find out which endpoint the ElectionStatusProvider calls, and then look up the response defined in MSW. Those steps are fine if you're working on this particular page, but get annoying if this page is just one part of some other investigation.